packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -338,14 +338,24 @@ export type ALElementTextOptions = Types.Options<
     maxDepth?: number;
     updateText?: <T extends ALElementText>(elementText: T, domSource: ALDOMTextSource) => void;
     getText?: <T extends ALElementText>(elementTexts: T[]) => ALElementText;
+    cacheText?: boolean;
```

Comment:
nit: Maybe rename to `initCacheText`, I initially got confused why there are two variables for using the text cache (also `skipCache`)